serverjs/updatecards.js

```diff
@@ -606,7 +606,7 @@ function convertCard(card, isExtra) {
     eur: card.prices.eur ? parseFloat(card.prices.eur, 10) : null,
     tix: card.prices.tix ? parseFloat(card.prices.tix, 10) : null,
   };
-  newcard.elo = catalog.elodict[name];
+  newcard.elo = catalog.elodict[name] ? catalog.elodict[name] : 1200;
```

Comment:
this can be written more concisely as `catalog.elodict[name] || 1200`

Reply:
True. I tried using `??` and that wasn't compatible, so I gave up. That brings up another question though - is it possible for a card to have 0 elo? As small of an edge case as it may be, it might warrant a `typeof` check instead.

Reply:
It's unrealistic for a card to have 0 Elo, `||` should be fine

Reply:
Okay, but when the Ghost of Bugs Yet to Come goes to haunt you at night, don't run to me.